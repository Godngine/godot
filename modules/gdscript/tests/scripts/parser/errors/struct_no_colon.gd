struct Test
    var oops

func test():
    pass
