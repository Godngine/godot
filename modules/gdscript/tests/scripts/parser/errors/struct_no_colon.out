GDTEST_PARSER_ERROR
Expected ":" after struct name.
